Olha_Lesiuk/HW_11/task_1.py

```diff
@@ -0,0 +1,273 @@
+# creating a parent class
+
```

Comment:
please read the task more carefully.  Every class should be in a separate module. please fix it after that I will review

Reply:
fixed, please check